examples/analog_hamiltonian_simulation/01_Introduction_to_Aquila.ipynb

```diff
@@ -7,175 +7,205 @@
     "tags": []
```

Comment:
<p>For plotting the phase, let's use <code>matplotlib.pyplot.step</code>, see here: https://github.com/aws/amazon-braket-examples/blob/ahs/examples/analog_hamiltonian_simulation/utils.py#L81</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>